src/__init__.py

```diff
@@ -25,33 +25,36 @@
 from mathutils import Vector
 from bpy.props import *
 
+from .import_properties import *
```

Comment:
This line give me "ImportError: attempted relative import with no known parent package"

Reply:
Check the merge, at the bottom is listed another file for the necessary import. Feel free to inline it. It's just something I use as a [standard toolset](https://github.com/LapisSea/auto_reproject/blob/master/import_properties.py) for my addons in order to clean up files that matter.

Reply:
ahhhhh, thanks. I did not see that.